Gems/EditorPythonBindings/Code/Source/ActionManager/PythonActionManagerHandler.h

```diff
@@ -8,33 +8,55 @@
 
 #include <EditorPythonBindings/CustomTypeBindingBus.h>
 #include <Source/ActionManager/ActionManagerBus.h>
+#include <Source/ActionManager/MenuManagerBus.h>
 
 namespace AzToolsFramework
 {
     class ActionManagerInterface;
+    class MenuManagerInterface;
 }
 
 namespace EditorPythonBindings
 {
-    class PythonEditorActionHandler final
+    class PythonActionManagerHandler final
         : public ActionManagerRequestBus::Handler
+        , public MenuManagerRequestBus::Handler
         , private EditorPythonBindings::CustomTypeBindingNotificationBus::Handler
     {
     public:
         using Handle = EditorPythonBindings::CustomTypeBindingNotifications::ValueHandle;
         using AllocationHandle = EditorPythonBindings::CustomTypeBindingNotifications::AllocationHandle;
         constexpr static Handle NoAllocation{ ~0LL };
 
-        PythonEditorActionHandler();
-        ~PythonEditorActionHandler();
+        PythonActionManagerHandler();
+        ~PythonActionManagerHandler();
+        
+        static void Reflect(AZ::ReflectContext* context);
 
         // ActionManagerPythonRequestBus overrides ...
-        ActionManagerOperationResult RegisterAction(
+        AzToolsFramework::ActionManagerOperationResult RegisterAction(
             const AZStd::string& contextIdentifier,
-            const AZStd::string& identifier,
+            const AZStd::string& actionIdentifier,
             const AzToolsFramework::ActionProperties& properties,
             PythonEditorAction handler) override;
-        ActionManagerOperationResult TriggerAction(const AZStd::string& actionIdentifier) override;
+        AzToolsFramework::ActionManagerOperationResult RegisterCheckableAction(
+            const AZStd::string& contextIdentifier,
+            const AZStd::string& actionIdentifier,
+            const AzToolsFramework::ActionProperties& properties,
+            PythonEditorAction handler,
+            PythonEditorAction updateCallback) override;
+        AzToolsFramework::ActionManagerOperationResult TriggerAction(const AZStd::string& actionIdentifier) override;
+        AzToolsFramework::ActionManagerOperationResult UpdateAction(const AZStd::string& actionIdentifier) override;
+        
+        // MenuManagerPythonRequestBus overrides ...
+        AzToolsFramework::MenuManagerOperationResult RegisterMenu(const AZStd::string& identifier,
+            const AzToolsFramework::MenuProperties& properties) override;
```

Comment:
uber nit: Is it possible to move the first param to the next line to be consistent with the other function declarations?